style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f0f0f0;
+  justify-content: center;
+  text-align: center;
+  margin-top: 300px;
+}
+
+.timer #Display {
```

Comment:
please change this also because the id name here and in your html file doesn't match.